vertica/datadog_checks/vertica/data/conf.yaml.example

```diff
@@ -148,10 +148,9 @@ instances:
     #     tags:
     #       - test:vertica
 
-    ## @param client_lib_log_level - string - optional
+    ## @param client_lib_log_level - string - optional - default: DEBUG
     ## Vertica library log level.
-    ## By default the vertica library logging is disabled.
-    ## Valid values: CRITICAL, FATAL, ERROR, WARN, WARNING, INFO, DEBUG
+    ## Valid values: CRITICAL, FATAL, ERROR, WARN, WARNING, INFO, DEBUG, null (disable logging).
```

Comment:
I don’t see any code change. You mean changing the default behaviour to be DEBUG or just suggest DEBUG as example in conf.yaml.example?

Reply:
:facepalm: Yup, forgot to change `vertica.py`! (It's Monday morning, right!) Anywya, good catch. 😄

Reply:
A possible improvement could be to use by default the same log level as the agent one